### PR TITLE
Package semver2.1.1.0

### PR DESCRIPTION
**Pull request by guyonvarch**
### `semver2.1.1.0`
Semantic version handling for OCaml

---
* Homepage: https://github.com/dividat/ocaml-semver
* Bug tracker: https://github.com/dividat/ocaml-semver/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/semver2/semver2.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "info@dividat.com"
+authors: ["Dividat AG"]
+synopsis: "Semantic version handling for OCaml"
+license: "MIT"
+homepage: "https://github.com/dividat/ocaml-semver"
+bug-reports: "https://github.com/dividat/ocaml-semver/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.2.0"}
```
Comment (mseri) [inline packages/semver2/semver2.1.1.0/opam:10]:
```suggestion
  "dune" {>= "1.2.0"}
```
Dune cannot be tagged as build dependency for forward compatibility reasons

Comment (mseri) [reply packages/semver2/semver2.1.1.0/opam]: Can you port this change upstream?

**Comment by kit-ty-kate** [comment]
Thanks!

File: packages/semver2/semver2.1.1.0/opam (hunk 1 above)
Comment (guyonvarch) [reply packages/semver2/semver2.1.1.0/opam]: Thanks for the modification!

Upstream change: https://github.com/dividat/ocaml-semver/commit/587a3c5696b8bcb973ec0fb1138d821239c7ce5a

**Pull request merged**